bundles/org.openhab.binding.paradoxalarm/src/main/resources/OH-INF/update/partition_type_update.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="paradoxalarm:partition">
+		<instruction-set targetVersion="1">
+			<add-channel id="detailedState">
+				<type>paradoxalarm:detailedState</type>
+				<label>Detailed Partition State</label>
+				<description>The detailed state of partition (contains sub-states like Stay in Armed, Armed no entry, etc)</description>
```

Comment:
Your file looks good.
You can remove label/description, they will be inherited.
What is missing is the "thingTypeVersion" property you need to add in the thing type (XML). Here is an example:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/washingMachine.xml#L47